AUDIO/SetRadioStationDisabled.md

```diff
@@ -0,0 +1,70 @@
+---
+ns: HUD
+aliases: N_0x94f2e83ead7e6b82
+---
+## _SET_RADIO_STATION_DISABLED
+
+```c
+// 0x94F2E83EAD7E6B82
+BOOL _SET_RADIO_STATION_DISABLED(char StationName, BOOL Toggle);
```

Comment:
`char`?

-> `char*`.

Also, please camelCase argument names, no first upper character.

Are you sure this returns a `BOOL`, too?